src/main/java/ua/utility/kfsdbupgrade/ForeignKeyReference.java

```diff
@@ -105,7 +105,13 @@ private String buildIndexName(int indx) {
 		StringBuilder retval = new StringBuilder(128);
 		retval.append(schemaName.toUpperCase());
 		retval.append(".");
-		retval.append(indexNameTemplate.replace("[table-name]", tableName).replace("{index}", "" + indx));
+		// Oracle before v 12.2 only allows identifiers upto 30 characters in length, so shorten index names
+		String shortTableName = tableName;
```

Comment:
I think this could be promlematic for the ORM in KFS. For example, if we dynamically shorten names in this project, there's a high liklihood that the related OJB mapping would need to be changed too. Thoughts?